project.clj

```diff
@@ -7,12 +7,21 @@
   :license {:name "Apache License, Version 2.0"
             :url "http://www.apache.org/licenses/LICENSE-2.0.html"}
 
-  :dependencies [[org.clojure/clojure "1.6.0"]
-                 [org.clojure/java.jdbc "0.3.2"]
-                 [org.postgresql/postgresql "9.3-1100-jdbc41"]
-                 [com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+  :dependencies [[com.jolbox/bonecp "0.8.0.RELEASE" :exclusions [[org.slf4j/slf4j-api]]]
+                 [com.zaxxer/HikariCP-java6 "2.3.7" :exclusions [[org.slf4j/slf4j-api]]]
+                 [metrics-clojure "2.5.1" :exclusions [[org.slf4j/slf4j-api]]]
```

Comment:
For everyone's ref, this is the registration code that @cprice404 is looking to centralize: https://github.com/puppetlabs/pe-puppet-server-extensions/blob/stable/src/clj/puppetlabs/enterprise/services/metrics/metrics_core.clj, plus some other namespaces around pe-puppet-server-extensions apply here.